recipes/gtk4/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+libX11
```

Comment:
Should we be using the conda-forge packages and/or possibly CDTs for these?

Reply:
Can these be replaced by CDTs (now)? I know they're here for the test phase to work and the [docs](https://conda-forge.org/docs/maintainer/knowledge_base.html#core-dependency-tree-packages-cdts) say to put things in `yum_requirements.txt` for that purpose, but it would definitely be cleaner otherwise.

Reply:
I replaced the CDTs with conda-forge equivalents and that seems to work. I also got rid of the `yum_requirements.txt` file.

Reply:
That swath of packages adds 13 MB to the download (tested locally), 11 MB of which is `mesalib`. I'm not entirely sure that the entire `mesalib` is required or what we want. But if we could trim that down it would be good.

Reply:
Also if we decide on depending on the xorg- packages here, it probably makes sense to do the same in `gtk3`.

Reply:
I'm happy either way, I'm happy to merge this once you have a consensus between yourselves.

According to the linking checks `mesalib` isn't being used so it can probably be dropped from run:

```log
WARNING (gtk4): dso library package conda-forge::xorg-libice-1.0.10-h7f98852_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): run-exports library package conda-forge::freetype-2.10.4-h0708190_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): dso library package conda-forge::xorg-libsm-1.2.3-hd9c2040_1000 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): dso library package conda-forge::mesalib-18.3.1-h590aaf7_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): run-exports library package conda-forge::xorg-libxinerama-1.1.4-h9c3ff4c_1001 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): dso library package conda-forge::xorg-libxtst-1.2.3-h516909a_1002 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
WARNING (gtk4): dso library package conda-forge::xorg-libxrender-0.9.10-h516909a_1002 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)
```

Reply:
Half the xorg stuff is getting installed with the `gtk3`-dependent programs already and no one has complained about performance issues, so I'm good with just moving to them over the CDTs here (we can think about doing the same with `gtk3` separately). That way the xorg dependencies will be pulled in by any packages that use `gtk4` and there will be no headaches about remembering to add the CDTs to those recipes.